integration/README.md

```diff
@@ -0,0 +1,27 @@
+# Integration tests
+
+Here we have a basic integration test for the happy path.
+
+## Requirements
+
+You need these tools installed locally:
+
+- [kind](https://kind.sigs.k8s.io/)
+- [apptestctl](https://github.com/giantswarm/apptestctl/)
+
+## Running the tests
+
+Create the `kind` cluster and use `apptestctl` to bootstrap GiantSwarm's App platform.
+
+```bash
+(kind delete cluster || true) && kind create cluster && apptestctl bootstrap --kubeconfig="$(kind get kubeconfig)"
+```
+
+You can now run the tests.
+
+```bash
+E2E_KUBECONFIG=~/.kube/config CIRCLE_SHA1=$(git rev-parse HEAD) AZURE_CLIENTID="${AZURE_CLIENTID}" AZURE_CLIENTSECRET="${AZURE_CLIENTSECRET}" AZURE_TENANTID="${AZURE_TENANTID}" AZURE_SUBSCRIPTIONID="${AZURE_SUBSCRIPTIONID}" go test -tags=k8srequired ./integration/test/... -count=1 | luigi
```

Comment:
Why not use $(kind get kubeconfig) here as well?

Reply:
`kind get kubeconfig` returns the contents of the config file, while the library we use for testing works with the path of the file.